aspnet/getting-started.rst

```diff
@@ -0,0 +1,53 @@
+Getting Started
+===============
+
+1. Install `.NET Core`_
+
+2. Create a new .NET Core project:
+
+  .. code-block:: console
+    
+    mkdir aspnetcoreapp
+    cd aspnetcoreapp
+    dotnet new
+
+3. Update your ``project.json`` file to add the package for the Kestrel HTTP server and the ASP.NET Core hosting model:
```

Comment:
You actually only add the Kestrel package as a direct dependency, which then pulls in a bunch of stuff. I'll simplify the text.